packages/fxa-settings/src/pages/PostVerify/SetPassword/en.ftl

```diff
@@ -0,0 +1,6 @@
+## SetPassword page
+## Third party auth users that do not have a password set yet are prompted for a
+## password to complete their sign-in when they want to login to a service requiring it.
+
+set-password-heading = Set your password
+set-password-info = Please create a password to continue to { -brand-firefox } Sync. Your data is encrypted with your password to protect your privacy.
```

Comment:
I didn't see "Firefox Sync" anywhere else as a brand surprisingly but I can add it to our brand list if we need to.

Reply:
Looks like it's otherwise only passed in as an (unlocalized) service name. I'd be in favour of setting it up as a brand/product term. :squint